game-core/src/main/java/org/triplea/game/server/AvailableGames.java

```diff
@@ -58,6 +49,16 @@
             availableGames.keySet().size(), availableGames.keySet()));
   }
 
+  private static Map<String, URI> getGames(final File map) {
+    log.info("Loading map: " + map);
+    if (map.isDirectory()) {
+      return getGamesFromDirectory(map);
+    } else if (map.isFile() && map.getName().toLowerCase().endsWith(ZIP_EXTENSION)) {
+      return getGamesFromZip(map);
+    }
```

Comment:
code-style side-note, I think we probably *should* be using 'else', it can prevent errors and secondly  it visually makes the logic block look more like a logic block.

If one were to be really consistent here the `else if` would be converted to an `if` statement as well.